doc/source/cluster/cluster_under_construction/ray-clusters-on-kubernetes/index.md

```diff
@@ -2,12 +2,32 @@
 (kuberay-index)=
 ## Overview
 
-You can execute your distributed Ray programs on a Kubernetes cluster.
+In this section we cover how to execute your distributed Ray programs on a Kubernetes cluster.
 
-The [KubeRay Operator](https://ray-project.github.io/kuberay/components/operator/) provides a Kubernetes-native
-interface for managing Ray clusters. Each Ray cluster consist of a head pod and collection of worker pods.
-Optional autoscaling support allows the KubeRay Operator to size your Ray clusters according to the requirements
-of your Ray workload, adding and removing Ray pods as needed.
+The [KubeRay Operator](https://ray-project.github.io/kuberay/components/operator/) is the recommended way to do so. It provides a Kubernetes-native way to manage Ray clusters. Each Ray cluster consist of
```

Comment:
nits:
`Using the Kuberay Operator is...`
`The KubeRay operator provides...`

Reply:
Fixing my own typo:
`Each Ray cluster consist` -> `Each Ray cluster consists`

Reply:
thanks, fixed